src/qt/clientmodel.cpp

```diff
@@ -30,14 +31,23 @@ ClientModel::ClientModel(interfaces::Node& node, OptionsModel *_optionsModel, QO
     optionsModel(_optionsModel),
     peerTableModel(nullptr),
     banTableModel(nullptr),
-    pollTimer(nullptr)
+    m_thread(new QThread(this))
 {
     cachedBestHeaderHeight = -1;
     cachedBestHeaderTime = -1;
     peerTableModel = new PeerTableModel(m_node, this);
     banTableModel = new BanTableModel(m_node, this);
-    pollTimer = new QTimer(this);
-    connect(pollTimer, &QTimer::timeout, this, &ClientModel::updateTimer);
+
+    pollTimer = new QTimer;
```

Comment:
Why can't this use `QTimer(m_thread)`?

Reply:
Because Qt threading model :trollface:

Being a child of QThread does nothing other than be owned by it. To have it's events processed in the thread then moveToThread must be used. This also works only if the object being moved has no parent.

Reply:
why can't this be a unique_ptr?

Reply:
I thought it deleted children when being deleted itself?

Reply:
You can't have a tree of objects with different thread affinity.

Reply:
> why can't this be a unique_ptr?

It can, but it requires more changes, which are unnecessary in the scope of this PR.